docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -91,19 +91,20 @@
 
     a {
       font-weight: hb-theme-font-weight(semibold);
+      position: relative;
```

Comment:
Is this to fix the icon issue in the external link? If so that is being fixed in #490.

Reply:
Yes. It was making me itch. LOL. I'll remove it. 👍